test/summary/reuse.js

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) SAP SE and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+
+const assert = require('assert')
+const summarizer = require('../../providers/summary/reuse')()
+
+describe('FsfeReuseSummarizer', () => {
+
+  it('should include populate all available attributes', () => {
+    const files = [{ fileName: 'README.md', licenseConcluded: 'MIT', licenseInfoFile: 'NOASSERTION', fileCopyrightText: 'Somebody', checksumSha1: '42' }, { fileName: 'SECURITY.md', licenseConcluded: 'NOASSERTION', licenseInfoFile: 'Apache-2.0', fileCopyrightText: 'Somebody else', checksumSha1: '23' }]
```

Comment:
nit: to make this more readable, could you split into multiple lines, similar to how it's done [here](https://github.com/clearlydefined/service/blob/80044b461a7d3c1b8bb92441f7a80f66e0d8d037/test/summary/licensee.js#L10-L14)?

```
    const files = [
        { fileName: 'README.md', licenseConcluded: 'MIT', licenseInfoFile: 'NOASSERTION', fileCopyrightText: 'Somebody', checksumSha1: '42' }, 
        { fileName: 'SECURITY.md', licenseConcluded: 'NOASSERTION', licenseInfoFile: 'Apache-2.0', fileCopyrightText: 'Somebody else', checksumSha1: '23' }
    ]

```

Reply:
Also similar for the other `files` variables below